redi-connect-front/src/i18n.tsx

```diff
@@ -0,0 +1,27 @@
+import i18n from 'i18next'
+import {
+  initReactI18next
+} from 'react-i18next'
+import Backend from 'i18next-xhr-backend'
+import LanguageDetector from 'i18next-browser-languagedetector'
+
+const languages = ['en']
+
+i18n
+  .use(Backend)
+  // detect user language
+  // learn more: https://github.com/i18next/i18next-browser-languageDetector
+  .use(LanguageDetector)
```

Comment:
is this really needed in our case?

Reply:
Not necessarily for one language but needed if we have more than 1 language so I'd just leave it there if that's ok for you